### PR TITLE
feat: implement blog page styles

### LINKED ISSUE #151: [Blog/News] Implement page styles
Implement overall styling for blog & news pages

- [ ] overall page width, margins and paddings
- [ ] style blog header (heading, subheader, main image)
- [ ] style blog sub headings within the text
- [ ] style blog links within the text
- [ ] sidebar section with bg color, section is feed by Target recommendations, can be empty

Blog post article info has a separate ticket #152, same for the social media banner #153

### PR DESCRIPTION
**Pull request by duynguyen**
Fix #151 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/how-cell-painting-making-mark
- After: https://151-blog-styles--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/how-cell-painting-making-mark

### CODE REVIEW

File: templates/blog/blog.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  const firstThreeChildren = Array.from(mainWrapper.children).slice(0, 3);
+  firstThreeChildren.forEach((child) => {
+    if (child.tagName === 'H1' && !blogH1) {
+      blogH1 = child;
+    } else if (child.tagName === 'P' && !blogHeroP1) {
+      blogHeroP1 = child;
+    } else if (child.tagName === 'P' && !blogHeroP2) {
+      blogHeroP2 = child;
+    }
```
Comment (rgravitvl) [inline templates/blog/blog.js:18]: Below page fails to render all the content. Above can go as static if check.
https://151-blog-styles--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/lab-tools-enhance-mrna-development-manufacturing-efficiency

**Review by rgravitvl — CHANGES_REQUESTED** [review]

File: templates/blog/blog.js (hunk 1 above)
Comment (duynguyen) [reply templates/blog/blog.js:18]: good spot @rgravitvl , fixed

**Comment by davenichols-DHLS** [comment]
Hi @duynguyen  and @rgravitvl .  Looking great, but I noticed that some pictures within the content are not having the correct ratio?  

Example:  https://151-blog-styles--danaher-ls-aem--hlxsites.hlx.page/us/en/blog/lab-tools-enhance-mrna-development-manufacturing-efficiency vs https://lifesciences.danaher.com/us/en/blog/lab-tools-enhance-mrna-development-manufacturing-efficiency.html

**Comment by duynguyen** [comment]
@davenichols-DHLS it seems to be from the columns block. created #183 to follow up.

**Review by davenichols-DHLS — APPROVED** [review]
Looks good - will add another task for updating small fixes.

**Review by rgravitvl — APPROVED** [review]

**Pull request merged**